reports/California.md

```diff
@@ -182,6 +200,20 @@ id: ca-longbeach-2
 
 ## Los Angeles
 
+### Police use tear gas and riot rounds against peaceful protestors | May 29th
+
+In footage taken by a bystander filming from her apartment, police advance down a street near a Downtown LA shopping center. Protestors retreat from police. Police continuously fire less-lethal rounds and chemical agents at protestors. Towards the end of the footage, a single firework is thrown at police by protestors.
```

Comment:
Probably should be non-lethal instead of less-lethal to match the language used in both sources

Reply:
I think non-lethal is a misnomer. Less-lethal or less-than-lethal is more the preferred term now, because while they are nowhere near the lethality of live rounds, they can still kill